### ISSUE TITLE
Add buttons to navigate playlists, jump to random video in playlist

### ISSUE THREAD

**Issue by tyzbit** [Tzahi12345/YoutubeDL-Material#376]

**Comment by Tzahi12345** [comment]
This should already happen, does it stop playing once the first video ends?

**Comment by tyzbit** [comment]
Yes, currently if you select a video that one video will play but when finished it won't proceed to the next video in the playlist.

**Comment by tyzbit** [comment]
The current nightly addresses watching videos in a playlist, so this issue now concerns adding a shuffle feature for watching playlists.

**Comment by dejan995** [comment]
> The current nightly addresses watching videos in a playlist, so this issue now concerns adding a shuffle feature for watching playlists.

@Tzahi12345 Maybe we can add this in a future release? It's a nice feature to have.
Also I would suggest adding the **_"enhancement"_** tag on this.

**Comment by GlassedSilver** [comment]
> 
> 
> > The current nightly addresses watching videos in a playlist, so this issue now concerns adding a shuffle feature for watching playlists.
> 
> @Tzahi12345 Maybe we can add this in a future release? It's a nice feature to have. Also I would suggest adding the **_"enhancement"_** tag on this.

done
